### PR TITLE
feat: Added support for Django 4.2

### PR DESCRIPTION
**Pull request by zubairshakoorarbisoft**
- Added support for Django 4.2
- Coded used https://github.com/adamchainz/django-upgrade

### CODE REVIEW

File: release_util/tests/test_check_reserved_keywords/test_app/test_app/settings.py [hunk 1]
```diff
@@ -103,7 +103,6 @@
 
 USE_I18N = True
 
-USE_L10N = True
```
Comment (awais786) [inline release_util/tests/test_check_reserved_keywords/test_app/test_app/settings.py:106]: Revert it

File: release_util/tests/migrations/test_migrations/0004_fourth.py [hunk 2]
```diff
@@ -0,0 +1,25 @@
+from django.db import migrations, models
```
Comment (awais786) [inline release_util/tests/migrations/test_migrations/0004_fourth.py:1]: These migrations are not of this ticket. Make them against master branch in separate PR.

File: release_util/tests/test_migration_commands.py [hunk 3]
```diff
@@ -220,13 +222,33 @@ def test_showmigrations_list(self):
...
                         {'app': 'release_util', 'migration': '0003_third'},
+                        {'app': 'release_util', 'migration': '0004_fourth'},
                     ]
                 },
                 exit_value=exit_code
             )
 
         call_command("migrate", "release_util", "0003", verbosity=0)
 
+        for fail_on_unapplied, exit_code in (
```
Comment (awais786) [inline release_util/tests/test_migration_commands.py:233]: revert this also.

File: release_util/tests/migrations/test_migrations/0004_fourth.py [hunk 4]
```diff
@@ -0,0 +1,28 @@
+# Generated by Django 4.2.3 on 2023-07-24 11:29
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
```
Comment (awais786) [inline release_util/tests/migrations/test_migrations/0004_fourth.py:6]: It was added due to https://docs.djangoproject.com/en/4.0/releases/4.0/#migrations-autodetector-changes

**Review by UsamaSadiq — CHANGES_REQUESTED** [review]
Please bump the package version and add CHANGELOG entry as well.

**Review by UsamaSadiq — APPROVED** [review]

File: release_util/tests/migrations/test_migrations/0004_fourth.py (hunk 4 above)
Comment (awais786) [reply release_util/tests/migrations/test_migrations/0004_fourth.py:6]: Als this is test migration nothing to do with prod.

**Pull request merged**